src/config/config.go

```diff
@@ -0,0 +1,5 @@
+package config
+
```

Comment:
on golang, containing source codes in "src" directory is deprecated because you need to specify src module every time you import modules via go modules.